app/src/main/res/layout-sw600dp-port/content_item.xml

```diff
@@ -50,8 +50,8 @@
     app:questionSplitViewPaddingTop="@{@dimen/space_0dp}"
     app:questionViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
     app:questionViewMarginBottom="@{@dimen/space_0dp}"
-    app:questionViewMarginEnd="@{@dimen/space_128dp}"
-    app:questionViewMarginStart="@{@dimen/space_128dp}"
+    app:questionViewMarginEnd="@{@dimen/content_item_question_view_margin_end}"
```

Comment:
Does this file only used for a Question or is there any other place where we are using this file?

If using at multiple places, we can go for more generic names here like `content_item_margin_end `

Reply:
In some files, we have two types of margin, one of them is exploration view and other is question view. So just to differentiate between those two I had to name these variables accordingly.

Reply:
@anandwana001 @ARJUPTA I agree with Arjun. We should use very specific names here considering that the UI for items related to exploration/question are really complex.